internal/cloudfront/aws_models.go

```diff
@@ -134,10 +137,25 @@ func newAWSOrigin(o Origin) *cloudfront.Origin {
 				Items:    SSLProtocols,
 				Quantity: aws.Int64(int64(len(SSLProtocols))),
 			},
-		},
-		DomainName: aws.String(o.Host),
-		Id:         aws.String(o.Host),
-		OriginPath: aws.String(""),
+		}
+		originAccessControlID = nil
+		s3OriginConfig = nil
```

Comment:
I think when `s3OriginConfig` is declared as a pointer is already nil, so I think is not needed to set. The same for `originAccessControlID` and `customOriginConfig` wdty?

Reply:
I did it more to make it explicit / help me visualize the changes / not forget anything 😄 . Removed 👍🏻 .